cassdegrees/ui/views.py

```diff
@@ -14,12 +14,20 @@ def index(request):
         {'url': "/api/model/degree/", 'img': "../static/img/create_plan_img.png", 'label': "Create Plan"},
         {'url': "/api/model/subplan/", 'img': "../static/img/create_subplan_img.png", 'label': "Create Subplan"},
         {'url': "", 'img': "../static/img/create_list_img.png", 'label': "Create List"},
-        {'url': "", 'img': "../static/img/open_existing_img.png", 'label': "Open Existing"},
+        {'url': "/list", 'img': "../static/img/open_existing_img.png", 'label': "Open Existing"},
         {'url': "/api/model/course/", 'img': "../static/img/manage_courses_img.png", 'label': "Manage Courses"}
     ]
 
     return render(request, 'index.html', context={'buttons': buttons})
 
+def planList(request):
+    # Generates a table based on whatever JSON object is stored in 'data'
+    # NOTE: data must be an array of dictionaries, with matching labels for each item
```

Comment:
This should be a doc comment, no? I believe the plan was to use PEP-8, which would describe that.